tests/ha/change_sbd_metadata.pm

```diff
@@ -0,0 +1,96 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Change SBD metadata and check result
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+=head1 NAME
+
+ha/change_sbd_metadata.pm - Change SBD metadata value, e.g. watchdog-time,
+allocate-timeout, msgwait-timeout, loop-timeout
+
+=head1 DESCRIPTION
+
+This module is used to check if metadata could be changed as expected.
+
+The key tasks performed by this module include:
+
+=over
+
+=item * Get the metadata configuration
+
+=item * Set each xxxx-timeout to $default_value + 2 on node01
+
+=item * Check if the changes affect on node01 and node02
+
+=item * Recover the metadata
+
+=back
+
+This includes the lock for multi-machine test.
+
+=over 
+
+=item * C<CLUSTER_BEFORE_CHANGE_METADATA_$cluster_name>
+
+=item * C<CLUSTER_AFTER_CHANGE_METADATA_$cluster_name>
+
+=back
+
+=cut
+
+use base 'haclusterbasetest';
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use hacluster;
+
+sub run {
+    my $cluster_name = get_cluster_name;
+
+    my $change_num = 2;
+    my %metadata_config;
+
+    # Get origin disk_metadata configuration
+    my $metadata_conf = script_output('crm sbd configure show disk_metadata');
+
+    # Output of crm sbd configure show :
+    # ==Dumping header on disk xxxxx
+    # Header version     : 2.1
+    # UUID               : xxxxxx-xxx-xxx-xx-xxxx
+    # Number of slots    : 255
+    # Sector size        : 512
+    # Timeout (watchdog) : 15
+    # Timeout (allocate) : 2
+    # Timeout (loop)     : 1
+    # Timeout (msgwait)  : 0
+    # ==Header on disk xxxxxxx
+    foreach my $line (split('\n', $metadata_conf)) {
+        if ($line =~ /Timeout\s+\((\w+)\)\s+\:\s+(\d+)/) {
+            $metadata_config{$1} = $2;
+        }
+    }
+
+    barrier_wait("CLUSTER_BEFORE_CHANGE_METADATA_$cluster_name");
```

Comment:
I think that thanks to `if (is_node(1))` on line 78, this BARRIER is not needed. Could you please elaborate about why is it needed?